R/internal.R

```diff
@@ -689,7 +689,7 @@ outputmatrix <- function(m, single.row, neginfstring, posinfstring,
                                        ci = ci
                         )$coefficients
                     } else { # significance from confidence interval
-                        if (is.numeric(ci.test) && !is.na(ci.test) && 
+                        if (is.numeric(ci.test) && !is.na(ci.test) && bold != .05 && 
```

Comment:
The `bold` argument can take other values than 0.05. If the user sets `bold = 0.01`, you would probably not want stars to be printed either, right? So I would do something like `bold == 0`.